src/pancam.py

```diff
@@ -0,0 +1,22 @@
+from follower import Follower
+
+follower = Follower()
+
+def seek():
+    #tag_data = follower.detect()
+    turn = follower.turn()
+
+    if follower.detect() == False:
+        print('Detect False')
+        follower._speed = follower.LEADER_MAX_SPEED
```

Comment:
Variables with leading underscore should be treated as private. It's not okay to write to a private member variable.